runtime/src/bank.rs

```diff
@@ -659,7 +659,8 @@ impl Bank {
         bank.finish_init();
 
         // Sanity assertions between bank snapshot and genesis config
-        // Consider removing from serializable bank state ([Ref]BankFields) and initializing
+        // Consider removing from serializable bank state
```

Comment:
Hello @ryoqun,
Can you please help me understand what is meant here? What specifically should we consider removing?